assets/src/scripts/components/hydrograph/index.js

```diff
@@ -296,9 +296,31 @@ const createTitle = function(elem) {
             elem.html(title);
         }, titleSelector));
 };
+/**
+ * Modify styling to hide or display the plot area.
+ *
+ * @param elem
+ * @param allTimeseries
+ */
+const controlGraphDisplay = function (elem, allTimeseries) {
+    const seriesWithPoints = Object.values(allTimeseries).filter(x => x.points.length > 0 && x.tsKey === 'current');
```

Comment:
You can at least partially filter the time series with the timeSeriesSelector('current') (in the timeseries module).

Reply:
Good idea! There are so many different selectors, I sometimes lose track of them. :flushed: